pkg/apis/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook/pkg/apis
 
-go 1.22.6
+go 1.22.0
```

Comment:
it should be 1.22.5 atleast due to csi-addons

Reply:
in that case we should need this pr right? @Madhu-1 as we are already in 1.22.6?

Reply:
Yes, we need to use 1.22.5 as the minimal required version

Reply:
> in that case we should need this pr right? @Madhu-1 as we are already in 1.22.6?

edit: `not` need